apps/condo/migrations/20220419095100-0123_manual_set_resident_in_ticket_client_field.js

```diff
@@ -0,0 +1,40 @@
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+            UPDATE "Ticket"
+            SET "client" = "Resident"."user",
+                "clientName" = "User"."name",
+                "clientPhone" = "User"."phone",
+                "clientEmail" = "User"."email"
+            FROM "Contact"
+            JOIN "User" ON
+                "Contact"."phone" = "User"."phone"
+            JOIN "Resident" ON
+                "Resident"."user" = "User"."id"
+            WHERE "Ticket"."client" IS NULL AND
+                  "Ticket"."contact" = "Contact"."id" AND
+                  "Resident"."property" = "Ticket"."property" AND
+                  "Resident"."unitName" = "Ticket"."unitName" AND
+                  "Resident"."unitType" = "Ticket"."unitType";
+        COMMIT;
+    `)
+}
+
+exports.down = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+            UPDATE "Ticket"
+            SET "client" = NULL
+            FROM "Contact"
+            JOIN "User" ON
+                "Contact"."phone" = "User"."phone"
+            JOIN "Resident" ON
+                "Resident"."user" = "User"."id"
+            WHERE "Ticket"."client" IS NOT NULL AND
```

Comment:
looks like this will disconnect all clients from tickets, I'm not sure that it is what we need here.

Reply:
this will disconnect clients from tickets, where client is resident user, which became connected during migration